CMake/macros.cmake

```diff
@@ -169,10 +165,9 @@ FUNCTION(QMC_RUN_AND_CHECK BASE_NAME BASE_DIR PREFIX INPUT_FILE PROCS THREADS SH
 
 
     SET( TEST_ADDED FALSE )
-    SET( TEST_LABELS "")
     SET( FULL_NAME "${BASE_NAME}-${PROCS}-${THREADS}" )
     MESSAGE("Adding test ${FULL_NAME}")
-    RUN_QMC_APP(${FULL_NAME} ${BASE_DIR} ${PROCS} ${THREADS} TEST_ADDED TEST_LABELS ${INPUT_FILE})
+    RUN_QMC_APP(${FULL_NAME} ${BASE_DIR} ${PROCS} ${THREADS} TEST_ADDED "integration" ${INPUT_FILE})
```

Comment:
TEST_LABELS is used on line 220 below and needs to be returned here.

Reply:
I just invoke ADD_TEST_LABELS